iroha/src/asset.rs

```diff
@@ -6,101 +6,103 @@ use parity_scale_codec::{Decode, Encode};
 pub struct Asset {
     /// identifier of asset, formatted as asset_name#domain_id
     pub id: Id,
-    pub amount: u128,
+    pub quantity: u128,
 }
 
 impl Asset {
     pub fn new(id: Id) -> Self {
-        Asset { id, amount: 0 }
+        Asset { id, quantity: 0 }
     }
 
-    pub fn with_amount(mut self, amount: u128) -> Self {
-        self.amount = amount;
+    pub fn with_quantity(mut self, quantity: u128) -> Self {
+        self.quantity = quantity;
         self
     }
 }
 
-pub mod isi {
-    use super::*;
-    use crate::isi::Contract;
-    use iroha_derive::{IntoContract, Io};
-    use parity_scale_codec::{Decode, Encode};
+/// Identification of an Asset. Consists of Asset's name, Domain's name and Account's name.
+///
+/// # Example
+///
+/// ```
+/// use iroha::asset::Id;
+///
+/// let id = Id::new("xor", "user", "company");
+/// ```
+#[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq, std::hash::Hash, Encode, Decode)]
+pub struct Id {
+    pub name: String,
+    pub container: String,
```

Comment:
Is `container` the same as `domain` as it's mentioned in docs? Actually in a few places word `container` is used in a few it's `domain`.

Reply:
https://soramitsu.atlassian.net/browse/RH2-88

Reply:
> Is `container` the same as `domain` as it's mentioned in docs? Actually in a few places word `container` is used in a few it's `domain`.

I'm not sure which docs you mean.